app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -507,6 +467,7 @@ const CallScreen = ({
                 {screenShareView}
                 {micPermissionsError && <PermissionErrorBar/>}
                 <EmojiList reactionStream={currentCall.reactionStream}/>
+                {showReactions && <ReactionBar raisedHand={myParticipant.raisedHand}/>}
```

Comment:
do we need another condition for those servers with a lower version of calls ?

Reply:
I think at this point we're not worrying about versioning pre-GA of calls -- correct me if I'm wrong @streamer45 . But if for some reason they don't, there just won't be any effect of sending an emoji.

Reply:
Yes. Worst case the server gets an "invalid ws event" log but that's about it.